swimlane/core/adapters/task.py

```diff
@@ -0,0 +1,54 @@
+from swimlane.core.resolver import SwimlaneResolver
+from swimlane.core.cache import check_cache
+from swimlane.core.resources.task import Task
+from swimlane.utils import one_of_keyword_only
+
+
+class TaskAdapter(SwimlaneResolver):
+    """Handles retreival of Swimlane Task Resources and execution of tasks against records."""
+    
+    @check_cache(Task)
+    @one_of_keyword_only('id', 'name')
+    def get(self, key, value):
+        """Get a single task by id or name"""
+        if key == 'id':
+            response = self._swimlane.request('get', 'task/{value}'.format(value=value))
```

Comment:
small nitpick can we name variables like they are in OpenApi -> `"task/{id}".format(id=value)`

Reply:
Sure, the only reason I named them this way was to prepare for a conversion to using fstrings so the variable can just be used by name in the string.

Reply:
Fixed in latest commits